apis/supernova/src/hooks.js

```diff
@@ -983,15 +983,11 @@ export function useOptions() {
 }
 
 /**
- * TODO before making public - expose getImperativeHandle on Viz
- * Exposes an API to the external environment.
- *
  * This is an empty object by default, but enables you to provide a custom API of your visualization to
  * make it possible to control after it has been rendered.
  *
  * You can only use this hook once, calling it more than once is considered an error.
  * @entry
- * @private
  * @template T
  * @param {function():T} factory
```

Comment:
maybe simplify to
```suggestion
 * @param {function():object} factory
```
The generic type T don't give any value

Reply:
hmm, you could have the handle function return whatever you want, shouldn't it be the generic type then? Not sure how a TS setup handles it?

Reply:
as this is the only place where the type is referenced I don't think it gives any value.
(but I also don't there is any real problem with keeping it as it is)